test_runner/fixtures/neon_fixtures.py

```diff
@@ -743,6 +743,9 @@ def ignore_postgres_log(path: str, _names):
         patch_script = ""
         for ps in self.env.pageservers:
             patch_script += f"UPDATE nodes SET listen_http_port={ps.service_port.http}, listen_pg_port={ps.service_port.pg}  WHERE node_id = '{ps.id}';"
+            # This is a temporary to get the backward compat test happy
```

Comment:
Is this because while the snapshot is from an AZ-aware version of the controller, we didn't add the neon_local support for AZs at that time?  i.e. prod databases will already have the AZ, it's just our neon_local-generated snapshots that don't?

Reply:
Exactly. Annoyingly, I reasoned through the order incorrectly ...

Reply:
ok -- I'm good with merging this if you can also do a spot-check on prod databases to make sure they definitely have AZs populated.